projects/driver_controls/src/driver_io.c

```diff
@@ -0,0 +1,83 @@
+#include <stddef.h>
+
+#include "adc.h"
+#include "gpio_it.h"
+#include "driver_io.h"
+
+// Arbitrary thresholds for gas pedal
+#define COAST_THRESHOLD 1000
+#define DRIVE_THRESHOLD 3000
+
+typedef struct DriverIOSettings {
+  GPIOAddress address;
+  GPIODir direction;
```

Comment:
Don't need to store all this if you're going to be setting them all the to the same thing later